app/kumactl/cmd/inspect/inspect_dataplanes_test.go

```diff
@@ -244,5 +244,19 @@ var _ = Describe("kumactl inspect dataplanes", func() {
 				Expect(testClient.receivedTags).To(HaveKeyWithValue("version", "v1"))
 			})
 		})
+
+		Describe("kumactl inspect dataplanes --gateway", func() {
+			It("gateway should be passed to the client", func() {
+				// given
+				rootCmd.SetArgs([]string{
+					"--config-file", filepath.Join("..", "testdata", "sample-kumactl.config.yaml"),
+					"inspect", "dataplanes", "--gateway"})
+
+				// when
+				err := rootCmd.Execute()
+				// then
+				Expect(err).ToNot(HaveOccurred())
+			})
+		})
```

Comment:
This tests only checks that the command is executed without an error. Can we add just add an assertion to the test above
`Expect(testClient.receivedGateway).To(BeTrue())`
and add receivedGateway in the testClient?